src/tests/test_integration/test_movies_models.py

```
@@ -10,8 +10,8 @@
     GenreModel,
     StarModel,
     DirectorModel,
-    CertificationModel,
 )
+from database.models.movies import CertificationModel, CertificationEnum
```

Ruff (0.11.9)

`database.models.movies.CertificationEnum` imported but unused

Remove unused import: `database.models.movies.CertificationEnum`

(F401)